src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -0,0 +1,770 @@
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+                },
+            })
+        }
+
+    @staticmethod
+    def getName():
+        return '3D Bladder 1'
+
+    @staticmethod
+    def getParameterSetNames():
+        return [
+            'Default',
+            'Cat 1',
+            'Rat 1']
+
+    @classmethod
+    def getDefaultOptions(cls, parameterSetName='Default'):
+        if 'Rat' in parameterSetName:
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Ostium Rat 1']
+        else:
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Ostium Cat 1']
+
+        options = {
+            'Number of elements up 1': 8,
+            'Number of elements up 2': 16,
+            'Number of elements around': 8,  # should be even
+            'Number of elements through wall': 1,
+            'Height': 5.0,
+            'Major diameter': 3.0,
+            'Minor diameter': 3.0,
+            'Urethra radius': 0.5,
+            'Bladder wall thickness': 0.05,
+            'Number of elements around ostium': 8,# implemented for 8
+            'Ureter': copy.deepcopy(ostiumOption),
+            'Ostium position around': 0.15,
+            'Ostium position up': 0.25,
+            'Number of elements radially on annulus': 1,
+            'Use cross derivatives': False,
+            'Refine': False,
+            'Refine number of elements around': 4,
+            'Refine number of elements up': 4,
+            'Refine number of elements through wall': 1
+        }
+
+        if 'Rat' in parameterSetName:
+            options = {
+                'Ureter': copy.deepcopy(ostiumOption),
+                'Number of elements up 1': 8,
+                'Number of elements up 2': 16,
+                'Number of elements around': 16,  # should be even
+                'Number of elements through wall': 1,
+                'Height': 3.0,
+                'Major diameter': 2.5,
+                'Minor diameter': 1.5,
+                'Urethra radius': 0.35,
+                'Bladder wall thickness': 0.05,
+                'Number of elements around ostium': 8,
+                'Ostium position around': 0.55,
+                'Ostium position up': 0.65,
+                'Number of elements radially on annulus': 2,
+                'Use cross derivatives': False
+            }
+        return options
+
+    @staticmethod
+    def getOrderedOptionNames():
+        return [
+            'Number of elements up 1',
+            'Number of elements up 2',
+            'Number of elements around',
+            'Number of elements through wall',
+            'Height',
+            'Major diameter',
+            'Minor diameter',
+            'Urethra radius',
+            'Bladder wall thickness',
+            'Ostium position around',
+            'Ostium position up',
+            'Ureter',
+            'Number of elements radially on annulus',
```

Comment:
Scaffoldmaker style is to put the 'Number' options together at the top of the list, 'Refine' at the bottom.
List each section in alphabetic order.